packages/core/src/impl/browser/FetchTransport.ts

```diff
@@ -143,6 +143,10 @@ export default class FetchTransport implements Transport {
   private async throwOnErrorResponse(response: Response): Promise<void> {
     if (response.status >= 300) {
       let text = ''
+      const headersToBeReturned: Record<string, string> = {}
+      response.headers.forEach((value, key) => {
+        headersToBeReturned[key] = value
+      })
```

Comment:
Is not necessary, please use our build-in function `getResponseHeaders(response)`:

```suggestion
```